pygmt/tests/baseline/test_coast_iceland.png.dvc

```diff
@@ -0,0 +1,4 @@
+outs:
```

Comment:
This file `pygmt/tests/baseline/test_coast_iceland.png.dvc` should be deletete.

Reply:
Ping @willschlitzer to address the comment and then this PR should be ready for merge.

Reply:
Done in 0a1c5e7